projects/core/styles/components/textfield.less

```diff
@@ -72,7 +72,7 @@ tui-textfield {
     input,
     select,
     textarea {
-        font: var(--tui-font-text-ui-m);
+        font: inherit;
```

Comment:
Is it expected behavior that placeholder inherits font from `[tuiFluidTypography]` ?

<img width="2016" height="128" alt="image" src="https://github.com/user-attachments/assets/b88eb097-cdfb-4541-a22c-6856136e5ec6" />

Reply:
Checked with designers and they said yeah